components/transaction/internal/adapters/postgres/operationroute/operationroute.postgresql.go

```
@@ -721,3 +743,35 @@ func (r *OperationRoutePostgreSQLRepository) FindTransactionRouteIDs(ctx context
+
+	for k, v := range fields {
+		trimmed := strings.TrimSpace(string(v))
+		if trimmed == "null" {
+			removeKeys = append(removeKeys, k)
+		} else {
+			merge[k] = v
+		}
+	}
+
+	if len(merge) > 0 {
+		mergeJSON, _ = json.Marshal(merge)
```

golangci-lint (2.11.3)

[error] Error return value of `encoding/json.Marshal` is not checked: unsafe type `encoding/json.RawMessage` found

(errchkjson)